### PR TITLE
Add metadata for app store submission

### PR DESCRIPTION
**Pull request by tsheng1**
### Added

- Images
- Licenses
- Icon
- General metadata

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- Updated metadata for submission to app store
+
+## [0.0.42] - 2021-03-25
```
Comment (btalma) [inline CHANGELOG.md:13]: It looks like this line was duplicated

**Review by btalma — COMMENTED** [review]
It looks like was a merge error in Changelog

File: CHANGELOG.md (hunk 1 above)
Comment (tsheng1) [reply CHANGELOG.md:15]: Fixed, thanks!

**Review by btalma — COMMENTED** [review]
I clicked too fast - no need to add the version number, it will be created when the branch is merged

**Comment by lucvysk** [comment]
Hmm! We might need to change the icon, guys! Better to use with the logo: https://www.accuratereviews.com/wp-content/uploads/2020/12/ShipStation.png (still need to improve the size, it's just a reference)

**Comment by lucvysk** [comment]
I also don't think this line is required: https://github.com/vtex-apps/shipstation/blob/master/manifest.json#L110
Might be better going to #app-store channel and ask for confirm

**Pull request merged**